.github/settings.yml

```diff
@@ -158,7 +158,7 @@ branches:
         # Required. Require branches to be up to date before merging.
         strict: true
         # Required. The list of status checks to require in order to merge into this branch
-        contexts: []
+        contexts: ['lint-and-test', 'action-semantic-pull-request']
```

Comment:
do we need to add this to the prototype repo too?

Reply:
yes we should!